easybuild/easyconfigs/s/SPLA/SPLA-1.4.0-CrayGNU-20.11-cuda-mkl.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'SPLA'
+version = '1.4.0'
+versionsuffix = '-cuda-mkl'
+
+homepage = 'https://github.com/eth-cscs/spla'
+description = """Specialized Parallel Linear Algebra"""
+
+toolchain = {'name': 'CrayGNU', 'version': '20.11'}
+toolchainopts = {'usempi': True, 'openmp': True, 'pic': True, 'verbose': False, 'opt': True}
+
+source_urls = ['https://github.com/eth-cscs/spla/archive']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [('CMake', '3.14.5', '', True)]
+
+dependencies = [
+  ('cudatoolkit', EXTERNAL_MODULE),
+  ('intel/19.1.1.217', EXTERNAL_MODULE),
```

Comment:
```suggestion
  ('intel', EXTERNAL_MODULE),
```

Reply:
If you are not bound to a specific Intel version, you can skip it and the default Intel compiler will be selected (in this case, the default is the version that you have selected)